apl-core/src/test/java/com/apollocurrency/aplwallet/apl/core/db/ReferencedTransactionServiceTest.java

```diff
@@ -41,13 +42,18 @@
 import org.mockito.Mockito;
 
 import javax.inject.Inject;
+import java.io.IOException;
+import java.nio.file.Path;
+
 @EnableWeld
 public class ReferencedTransactionServiceTest {
     @Inject
     private JdbiHandleFactory jdbiHandleFactory;
 
     @RegisterExtension
-    static DbExtension extension = new DbExtension();
+    DbExtension extension = new DbExtension(DbTestData.getDbFileProperties(createPath("targetDb").toAbsolutePath().toString()));
```

Comment:
File db is not required here

Reply:
No, it's required here. Otherwise two test methods fail with error:

A file path that is implicitly relative to the current working directory is not allowed in the database URL "jdbc:h2:file:**null**/apl-blockchain-shard-0000001;;MV_STORE=FALSE;CACHE_SIZE=262144". Use an absolute path, ~/name, ./name, or the baseDir setting instead. [90011-196]